<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrescriptionLabel</class>
 <widget class="QFrame" name="PrescriptionLabel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>988</width>
    <height>583</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #FFFFFF</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Plain</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="l1s1Label">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>18</pointsize>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="text">
        <string notr="true">LINE 1 SECTION 1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="l1s2Label">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>18</pointsize>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="text">
        <string notr="true">LINE 1 SECTION 2</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="l2s1Label">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>18</pointsize>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="text">
        <string notr="true">LINE 2 SECTION 1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="l2s2Label">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>18</pointsize>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="text">
        <string notr="true">LINE 2 SECTION 2</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="lastLabel">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>28</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="text">
        <string notr="true">REALLYLONGLASTNAME</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_23">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>24</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="text">
        <string notr="true">,</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="firstLabel">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>28</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="text">
        <string notr="true">REALLYLONGFIRSTNAME</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>18</pointsize>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="text">
        <string notr="true">Written:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="writtenLabel">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>18</pointsize>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="text">
        <string notr="true">12/23/2012</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>18</pointsize>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="text">
        <string notr="true">Filled:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="filledLabel">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>18</pointsize>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="text">
        <string notr="true">12/23/2012</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="prescriberLabel">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>18</pointsize>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="text">
        <string notr="true">Dr. Evans, Douglas E.</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>18</pointsize>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="text">
        <string notr="true">Filled by:</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="fillerLabel">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>18</pointsize>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="text">
        <string notr="true">DE</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="medicationLabel">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>28</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="text">
        <string notr="true">REALLYLONGMEDICATION</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QLabel" name="formLabel">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>28</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="text">
        <string notr="true">FORM</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="strengthLabel">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>28</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="text">
        <string notr="true">STRENGTH</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="genericHeaderLabel">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>18</pointsize>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="text">
        <string notr="true">Generic for</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="genericLabel">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>18</pointsize>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="text">
        <string notr="true">REALLYLONGMEDICATION</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>18</pointsize>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="text">
        <string notr="true">MFG:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="manufacturerLabel">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>18</pointsize>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="text">
        <string notr="true">LONGMANUFACTURER</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>28</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="text">
        <string notr="true">Expiration:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="expireLabel">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>28</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="text">
        <string notr="true">12/23/2014</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>18</pointsize>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="text">
        <string notr="true">NDC:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="ndcLabel">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>18</pointsize>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="text">
        <string notr="true">55555-55555</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="label_11">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>18</pointsize>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="text">
        <string notr="true">Dispensed:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="dispensedLabel">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>18</pointsize>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="text">
        <string notr="true">8888888</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="instructionsLabel">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>28</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string notr="true">INSTRUCTIONS INSTRUCTIONS INSTRUCTIONS INSTRUCTIONS INSTRUCTIONS INSTRUCTIONS INSTRUCTIONS INSTRUCTIONS </string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
